bin/autodetect/CCmdLineParser.cc

```diff
@@ -118,7 +117,7 @@ bool CCmdLineParser::parse(int argc,
             ("multivariateByFields",
                         "Optional flag to enable multi-variate analysis of correlated by fields")
             ("multipleBucketspans",  boost::program_options::value<std::string>(),
-                        "Optional comma-separated list of additional bucketspans - must be direct multiples of the main bucketspan")
+                        "Deprecated - ignored")
```

Comment:
what's the reason to keep it? Are there clients using this?

Reply:
I'm happy to completely remove this, although it does then commit us to removing the corresponding options on the Java side.  @dimitris-athanasiou said he would do this.  The functionality to silently drop any settings related to this functionality (which was never documented and not fully tested) will then live in the Java code.

Reply:
I think it's fine to drop everything from the c++ side. I'll prepare the java side and we will merge them both in `6.5`.

Reply:
I raised https://github.com/elastic/elasticsearch/pull/32496. @tveasey, this means you can completely remove the `multipleBucketspans` param in this PR.

Reply:
Ok great. I'll tidy.